docs/ldml/tr35.md

```diff
@@ -1312,14 +1316,20 @@ Examples:
 <tr><td colspan="3">…</td></tr>
 </tbody></table>
 
+ _However, if users typically expect a their language in non-default script to contain an admixture of another language, then the -t- and hybrid subtags may be omitted. Thus if Hindi written in Latin script normally contains an admixture of English text, ‘hi-Latn’ can be used instead of ‘hi-Latn-t-en-h0-hybrid’._
 > _Note: The [unicode_language_id](#unicode_language_id) should be the language used as the ‘scaffold’: for the fallback locale for internationalization services, typically used for more of the core vocabulary/structure in the content. Thus Hinglish should be represented as hi-t-h0-en where Hindi is the scaffold, and as en-t-h0-hi where English is._
 
-The value of -t- is a full _[unicode_language_id](#unicode_language_id)_, and can contain subtags for script or region where it is important to include them, as in the following. It may be useful in order to emphasize the script, even where it is the default script for the language, if it is not the same as the script of the main language tag.
+The value of -t- is a full _[unicode_language_id](#unicode_language_id)_, and can contain subtags for region where it is important to include it, as in the following. The value can also include the script, although that is is not normally included: the only instance where it should be is where the content of the source text varies by script. So because zh-Hant has different vocabulary and expressions, it could make sense to have en-t-zh-hant to make that distinction.
+
+> Note: The default script for the language is computed without reference to the hybrid subtags. Thus the default script for 'ru' is “Cyrl”, no matter what the source is in the -t- tag.
 
 <!-- HTML: no header -->
 <table><tbody>
-<tr><td>ru-t<u>-en-latn-gb-h0-hybrid</u></td>   <td>Runglish</td>   <td>Russian with an admixture of British English in Latin script</td></tr>
-<tr><td>ru-t-<u>en-cyrl-gb-h0-hybrid</u></td>   <td>Runglish</td>   <td>Russian with an admixture of British English in Cyrillic script</td></tr>
+<tr><td>ru-t-<u>en-h0-hybrid</u></td>   <td>Runglish</td>   <td>Russian with an admixture of American English</td></tr>
+<tr><td>ru-t-<u>en-gb-h0-hybrid</u></td>   <td>Runglish</td>   <td>Russian with an admixture of British English</td></tr>
+<tr><td>ru-Latn-t<u>-en-gb-h0-hybrid</u></td>   <td>Runglish</td>   <td>Russian with an admixture of British English in Latin script</td></tr>
+<tr><td>en-t-<u>zh-h0-hybrid</u></td>   <td>Chinglish</td>   <td>American English with an admixture of Chinese (Mandarin, China)</td></tr>
+<tr><td>en-t-<u>zh-Hant-h0-hybrid</u></td>   <td>Chinglish</td>   <td>American English with an admixture of Chinese (Mandarin, Taipai)</td></tr>
 </tbody></table>
```

Comment:
What is "Taipai" supposed to be here?

Reply:
Submitted PR: https://github.com/unicode-org/cldr/pull/1860